components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -45,13 +45,15 @@
 #  define bfb_log(val) cxx_log(val)
 #  define bfb_log10(val) cxx_log10(val)
 #  define bfb_exp(val) cxx_exp(val)
+#  define bfb_sqrt(val) bfb_pow(val,0.5_rtype)
```

Comment:
See `#  define bfb_cbrt(base) cxx_cbrt(base)`, line 43, for an example of how to handle this case. Search for `cxx_cbrt` in the repo to see how to wire up `cxx_sqrt`.